internal/tweets/tweets.go

```diff
@@ -372,3 +373,78 @@ func GetFollowingTweets(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	json.NewEncoder(w).Encode(tweets)
 }
+func Explore(w http.ResponseWriter, r *http.Request) {
```

Comment:
that's the opposite of what it's supposed to do. This one supposed to be everything *except* own tweets and the ones that user follows